test/call.c

```diff
@@ -1203,15 +1265,15 @@ int test_call_change_videodir(void)
 	err = module_load(".", "fakevideo");
 	TEST_ERR(err);
 
-	f->behaviour = BEHAVIOUR_PROGRESS;
+	f->behaviour = BEHAVIOUR_NOTHING;
 	f->estab_action = ACTION_NOTHING;
 
 	/* Make a call from A to B */
 	err = ua_connect(f->a.ua, 0, NULL, f->buri, VIDMODE_ON);
 	TEST_ERR(err);
 
 	/* wait for CALL_PROGRESS */
-	err = re_main_timeout(10000);
+	err = re_main_timeout(5000);
```

Comment:
I would prefer to skip this change.

The reason is that sometimes the test need close to 10 seconds to complete.
For example when running with with valgrind or qemu, and multiple parallel builds.

Reply:
Okay, the reason was only because of faster debugging. I remove all of this changes in this PR.